introducing-metrics-in-innersource.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Introducing Metrics in the InnerSource Initiative
+
+# Context
+
+An organization is applying InnerSource. This needs to check the
+current status and if some of the expected outcomes are starting
+to work.
+
+
+# Problem
+
+This organization may already have qualitative feedback from the
+involved teams, but desire more objective information focused
+on development activities.
+
+The organization does not really know where to start measuring
+things or what are the key parameters to measure.
+
+
+# Forces
+
+People do not like to be tracked or measured.
+
+There is not a proper monitoring infrastructure for the software
+development process and this is hard to build or to get funding
+for this.
+
+There is not a culture of software development metrics.
+
+Metrics are usually misunderstood if people have not received any
+training on those.
+
+
+# Solution
+
+Bring developers, middle managers and C-level to have a discussion
+about metrics.
```

Comment:
> 🙋  Suggested edit

Bring developers, middle managers and C-level to have a discussion about business objectives and metrics to track those objectives.

Reply:
We're working on it, @gruetter. We have not formalized anything, yet, but we plan on:

1. Auto-generating a social graph for "communities of interest" based on Git repository commits by categories defined within [TechRadar Languages & Technologies](https://www.thoughtworks.com/radar/a-z) to segment our internal engineering social network.
1. Using [architecture decision records (ADRs)](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions) to encourage discussions about enterprise standards.
1. Engaging engineering first to generate metrics ADRs about InnerSource metrics.

We're using a "man-on-the-moon" analogy to encourage everyone to thing about metrics as "stages." The metrics collected and reported are different for "launch," "lunar orbit," "lunar landing," etc.

Reply:
I very much like the idea of staging and I can imagine that using ADRs (never heard of these before, thanks for the reference ;) can indeed be helpful as crystallization points for discussions. I'm not sure I got your idea wrt. the _social graph_, though.